CDMS_VCS/avg_cloudiness.py

```diff
@@ -0,0 +1,35 @@
+import vcs, cdms2, sys, os
+
+file = cdms2.open(os.path.join(vcs.sample_data, "clt.nc"))
+clt = file("clt", time=('1985-1', '1986-1'))
```

Comment:
It looks like the selectors are inclusive, not exclusive, so you actually want to do `("1985-1", "1985-12")` (my bad!)